warn that ec2 join requires empty data dir (#14379)
gravitational/teleport: docs/pages/setup/guides/joining-nodes-aws-ec2.mdx

- An AWS EC2 instance to act as a Teleport node, with the Teleport binary
  installed.
  

> - AWS EC2 node should not have an existing data dir (e.g. `/var/lib/teleport`). 

Reviewer: ```suggestion
The Node should not have an existing data dir (`/var/lib/teleport` by default). 
```

> - AWS EC2 node should not have an existing data dir (e.g. `/var/lib/teleport`). 

Author: addressed @ptgott

>   Remove if this had a previous joining method.

Reviewer 2: ```suggestion
  Remove if this instance has previously joined a Teleport cluster.
```

- An AWS EC2 instance to act as a Teleport node, with the Teleport binary
  installed.
  

> - AWS EC2 node should not have an existing data dir (e.g. `/var/lib/teleport`).

Author: addressed @ptgott

>   Remove if this had a previous joining method.

Reviewer 2: ```suggestion
  Remove if this instance has previously joined a Teleport cluster.
```

## Step 4/4. Launch your Teleport node

<Admonition
  type="note"
  title="IMPORTANT"

> >

Reviewer 2: I think this tag in unclosed

Author: Thanks for reviewing, it was a lint issue.  There is a tag to close this.

## Step 4/4. Launch your Teleport node

<Admonition
  type="note"
  title="IMPORTANT"
>

>   The data directory (`/var/lib/teleport` by default) should be empty prior to launching the

Reviewer: ```suggestion
  The data directory (`/var/lib/teleport` by default) must be empty prior to launching the
```

>   node.  An error will occur if this node had previously joined by another method (e.g. token or IAM) and is now switching to the EC2 method. 

Reviewer: ```suggestion
  Node. An error will occur if this Node had previously joined by another method (e.g. token or IAM) and is now switching to the EC2 method. 
```

>   node.  An error will occur if this node had previously joined by another method (e.g. token or IAM) and is now switching to the EC2 method. 

Reviewer: What is the nature of the error? Will running `teleport start` fail with a message, or will this be something more mysterious?

Author: @ptgott modified phrasing

## Step 4/4. Launch your Teleport node

<Admonition
  type="note"
  title="IMPORTANT"
>

>   The data directory used (e.g. `/var/lib/teleport` by default) should be empty prior to launching
>   node.  An error will occur if this node had attached by another method, such as IAM and now switching to
>   EC2 Joining. 

Reviewer 2: ```suggestion
  The data directory (`/var/lib/teleport` by default) should be empty prior to launching the
  node.  An error will occur if this node had previously joined by another method (e.g. token or IAM) and is now switching to the EC2 method. 
```

>   </Admonition>

Reviewer 2: ah this closing tag is unhighlighted in GH because it's indented (and the opener isn't)
```suggestion
</Admonition>
```
